pkg/migrations/sql/preinstall/003-tables.sql

```diff
@@ -99,8 +99,11 @@ CREATE TABLE SCHEMA_CATALOG.metric (
     retention_period INTERVAL DEFAULT NULL, --NULL to use the default retention_period
     default_compression BOOLEAN NOT NULL DEFAULT true,
     delay_compression_until TIMESTAMPTZ DEFAULT NULL,
-    UNIQUE (metric_name) INCLUDE (table_name),
-    UNIQUE(table_name)
+    table_schema name NOT NULL DEFAULT 'SCHEMA_DATA',
```

Comment:
shouldn't we put this as the third column, right before table_name?

Reply:
I've done that before but had issues with upgrade tests reporting different column ordering for fresh install vs. upgrading.

Changing the test to account for different column ordering doesn't seem a trivial thing to me (might be wrong).